### PR TITLE
added manifest changes for nodejs samples

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Mohammed-MSFT — APPROVED** [review]
Approved, Changes looks fine.

**Review by Jegadeesh-MSFT — APPROVED** [review]
changes look good.

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6176427767

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6219931794

File: samples/meeting-recruitment-app/nodejs/.gitignore [hunk 1]
```diff
@@ -26,7 +26,7 @@ yarn-error.log*
 env/.env.*.user
 env/.env.local
 .localConfigs
-appPackage/build
+appMAnifest/build
```
Comment (ChetanSharma-msft) [inline samples/meeting-recruitment-app/nodejs/.gitignore:29]: appManifest

File: samples/tab-sso/nodejs/README.md [hunk 2]
```diff
@@ -109,8 +109,8 @@ Your tab needs to run as a registered Azure AD application in order to obtain an
...
     * Replace `{your_tunnel_domain}` with the subdomain you've assigned to your Ngrok account in step #1 above.
     * Edit the `manifest.json` for `webApplicationInfo` resource `"api://{your_tunnel_domain}/{{AppId}}"` with MicrosoftAppId. E.g. `"api://1245.ngrok-free.app/{{AppId}}`.
-    **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `tab-sso\nodejs\appManifest_Hub` folder with the required values.
-    **Zip** up the contents of the `appManifest` folder to create a `Manifest.zip` or `appManifest_Hub` folder to create a `appManifest_Hub.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+    **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `tab-sso\nodejs\Manifest_Hub` folder with the required values.
```
Comment (ChetanSharma-msft) [inline samples/tab-sso/nodejs/README.md:112]: Please check these:
image

File: samples/tab-stage-view/nodejs/Readme.md [hunk 3]
```diff
@@ -126,8 +126,8 @@ Also update `BaseUrl` according to your code runtime environment.
    - `<<BASE-URL>>` with your application's base url, e.g. https://1234.ngrok-free.app
    - `<<YOUR-MICROSOFT-APP-ID>>` with the `MicrosoftAppId` received from AAD app registration in Azure portal.
    - `<<DOMAIN-NAME>>` with the ngrok URL or app hosted base url.
-   **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `tab-stage-view\nodejs\appManifest_Hub` folder with the required values.
-   - **Zip** up the contents of the `appManifest` folder to create a `Manifest.zip` or `appManifest_Hub` folder to create a `appManifest_Hub.zip`
+   **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `tab-stage-view\nodejs\Manifest_Hub` folder with the required values.
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/nodejs/Readme.md:129]: Same here
image

File: samples/tab-staggered-permission/nodejs/README.md [hunk 4]
```diff
@@ -144,16 +144,16 @@ Make sure you've downloaded and installed Ngrok on your local machine. ngrok wil
...
     -  **Edit** the `manifest.json` for `webApplicationInfo` resource `"api://{{domain-name}}/{{Microsoft-App-Id}}"` with MicrosoftAppId. E.g. `"api://ngrok-free.app/00000-0000-0000"`
     - **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `tab-staggered-permission\nodejs\appManifest_Hub` folder with the required values.
-    - **Zip** up the contents of the `Manifest` folder to create a `Manifest.zip` or `appManifest_Hub` folder to create a `appManifest_Hub.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+    - **Zip** up the contents of the `appManifest` folder to create a `Manifest.zip` or `appManifest_Hub` folder to create a `Manifest_Hub.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
```
Comment (ChetanSharma-msft) [inline samples/tab-staggered-permission/nodejs/README.md:151]: This also looks wrong.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
added more review comments

**Comment by Prithvi-MSFT** [comment]
Added changes inside below PR
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/935#pullrequestreview-1630610155

File: samples/tab-staggered-permission/nodejs/README.md (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/tab-staggered-permission/nodejs/README.md:151]: changed

File: samples/tab-stage-view/nodejs/Readme.md (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/tab-stage-view/nodejs/Readme.md:129]: changed

File: samples/tab-sso/nodejs/README.md (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/tab-sso/nodejs/README.md:112]: changed

File: samples/meeting-recruitment-app/nodejs/.gitignore (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/meeting-recruitment-app/nodejs/.gitignore:29]: changed

**Pull request merged**